src/ast/ast_validations.c

```diff
@@ -1221,6 +1221,7 @@ static AST_Validation _ValidateQueryTermination(const AST *ast) {
 	   type != CYPHER_AST_MERGE    &&
 	   type != CYPHER_AST_DELETE   &&
 	   type != CYPHER_AST_SET      &&
+	   type != CYPHER_AST_REMOVE   &&
```

Comment:
Please update comment above:
`// (CREATE, MERGE, DELETE, SET, or REMOVE once supported)`
Remove is supported.